### ISSUE TITLE
Using 3rd party app-switcher (alt+tab) stops files being drag and dropped

### ISSUE THREAD

**Issue by UrbanBreznik** [files-community/Files#15971]
### Description

I am using a 3rd party program called Alt-Tab Terminator for switching between apps. When I try dragging a file from Files to any other app using this interface, it looses the files (does not drag them any more)

### Steps To Reproduce

1. Need alt-tab terminator (free version has the same bug) https://www.ntwind.com/software/alttabter.html
2. In ALt-tab Terminator selectd to replace the alt-tab with this program
3. Try draging a file from Files to any program switching with alt-tab (Alt-Tab terminator interface)

### Requirements

The expected behaviour is that it does not loose the files. If i Do this with file explorer the files stay at my cursor and i can drop them in a selected app.

### Files Version

3.6.0.0

### Windows Version

10.0.22631.3958

### Log File

[debug.log](https://github.com/user-attachments/files/16522760/debug.log)

**Comment by Josh65-2201** [comment]
@UrbanBreznik Is this still an issue on v4.0.11 https://files.community/download

**Comment by Josh65-2201** [comment]
@UrbanBreznik Can you answer the above question

**Comment by UrbanBreznik** [comment]
Yes, still the same! Needed to switch to a different file manager :(

**Comment by Josh65-2201** [comment]
Thanks for checking
